util/task/ticker_task_test.go

```diff
@@ -0,0 +1,62 @@
+package task_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/prebid/prebid-server/util/task"
+	"github.com/stretchr/testify/assert"
+)
+
+type MockRunner struct {
+	RunCount int
+}
+
+func (mcc *MockRunner) Run() error {
+	mcc.RunCount++
+	return nil
+}
+
+func TestStartWithSingleRun(t *testing.T) {
+	// Setup:
+	runner := &MockRunner{RunCount: 0}
+	interval := 0 * time.Millisecond
+	ticker := task.NewTickerTask(interval, runner)
```

Comment:
`TestStop` does make a `ticker.Stop()` call. Are we missing that call here?

Reply:
We shouldn't need it here because when `Start` is called, the interval is zero so the go routine the instantiates the ticker is never started. Since it never starts, there is no need to set a signal on the `done` channel because we don't need to stop the ticker.
In `TestStop`, we are testing the `Stop` method which is why it is called. Also, in that test the go routine starts and the ticker is instantiated.

Reply:
This is perhaps a bit of overkill for the tests, but if we think it would prevent issues then you have my support. This brings up a good thought though, are we probably handling graceful shutdowns? When PBS receives a quit signal from the OS, are we stopping the fetchers? If not, that might be a good follow-up PR.

Reply:
I can open a story to ensure we are gracefully shutting down.

Reply:
I have seen the shutdown pattern in fetchers and I was thinking that since those goroutines maintain no external state to gracefully sync, are there any drawbacks to just letting them be terminated ? Their connections are going to be closed when the process ends anyway, and that seems to be the only time this happens. We don't reload configs, where this could be useful.

Reply:
Perhaps it might not be needed. We'll created an internal story to check if there's a need for a graceful shutdown.